data/snippet/signed_request.toml

```diff
@@ -0,0 +1,49 @@
+id = "signed_request"
+name = "Signed Request/Response"
+description = "Check signatures of requestst and sign responses with a private key"
+code = """
+const crypto = require('crypto')
```

Comment:
What do you think of adding a tag to the snippets
"ESM" with hover over "Requires ES Modules, so cannot view in previewer without webpack" and then in the demo link we can have the unminifed webpacked snippet code with the important code on top

Without something specifying this requires webpack it technically should be a boilerplate instead, but I actually prefer as a snippet also. Just need a way to communicate the difference

Reply:
Note just curious of your opinion I can actually implement it if you like

Reply:
That sounds good to me.